rotary/rules/rotary.smk

```diff
@@ -300,25 +324,25 @@ rule qc_long_length_stats:
 
 rule qc_long:
     input:
-        "stats/qc_long_length_stats.txt"
+        expand("{sample}/stats/qc_long_length_stats.txt",sample=SAMPLE_NAMES)
     output:
         temp(touch("checkpoints/qc_long"))
 
 
 rule assembly_flye:
     input:
-        "qc_long/nanopore_qc.fastq.gz"
+        "{sample}/qc_long/nanopore_qc.fastq.gz"
     output:
-        "assembly/flye/assembly.fasta",
-        "assembly/flye/assembly_info.txt"
+        assembly="{sample}/assembly/flye/assembly.fasta",
+        info="{sample}/assembly/flye/assembly_info.txt",
+        output_dir=directory("{sample}/assembly/flye")
```

Comment:
Now that the path contains a wildcard, I support this move of output directory paths to the `output` field -- this is better practice I think. Similarly, thanks for moving the short read input files (in other rules) from the params to the input field